.husky/pre-commit

```diff
@@ -5,4 +5,8 @@ node ./scripts/flows.js
 npm run generate:tests --pre_commit=true
 git add ./integrations/*/tests/*.test.ts
 git add ./flows.yaml
+
+npm run generate:readmes
+git add ./integrations/**/*.md
+
```

Comment:
If you want to commit in this repo you must be patient ahah

Reply:
Yeah, this was why I wanted it to be in an Action originally. In general I'm not sure adding pre-commit as a place for logic is a healthy practice.